### PR TITLE
Move error-rates app into demos repo

### PR DESCRIPTION
**Pull request by lucasponce**
Moving the test-error-rates app being used to test kiali health config scenarios to the kiali/demos repo.

### CODE REVIEW

File: error-rates/README.adoc [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+A set of clients and servers that simulate different error requests.
+
+== Platform Install
+
+This demo has been tested using https://istio.io/latest/docs/setup/platform-setup/minikube/[Minikube] and https://istio.io/latest/docs/setup/install/istioctl/#install-a-different-profile[Istio 1.6 Demo Profile]
+
+== Quick Start
+
+Create `alpha` and `betha` namespaces. Add `istio-injection` label and deploy demo app.
```
Comment (jshaughn) [inline error-rates/README.adoc:11]: trivial (no change required) just a note than in English it's 'beta' as opposed to 'betha'.

File: error-rates/README.adoc [hunk 2]
```diff
@@ -0,0 +1,80 @@
+= Error Rates Demo
+
+A set of clients and servers that simulate different error requests.
+
+== Platform Install
+
+This demo has been tested using https://istio.io/latest/docs/setup/platform-setup/minikube/[Minikube] and https://istio.io/latest/docs/setup/install/istioctl/#install-a-different-profile[Istio 1.6 Demo Profile]
+
```
Comment (jshaughn) [inline error-rates/README.adoc:8]: Maybe a note that this requires, `go`, `Docker`, `kubectl` and `istioctl`.

File: error-rates/README.adoc (hunk 1 above)
Comment (lucasponce) [reply error-rates/README.adoc:11]: ah, good one.

File: error-rates/README.adoc (hunk 2 above)
Comment (lucasponce) [reply error-rates/README.adoc:8]: No, it's consistent with the other demo.
It requires just Minikube and Istio for basic usage, I don't want to detail more the "how to build".

**Pull request merged**